bank/src/main/java/org/example/account/Account.java

```diff
@@ -0,0 +1,115 @@
+package org.example.account;
+
+
+import java.math.BigDecimal;
+
+
+public class Account {
+
+    public static int howmanyaccout = 0;
```

Comment:
```suggestion
    private static int howmanyaccout = 0;
```
접근제어자를 private로 하여 캡슐화를 지키도록 합시다.
Account를 관리하고 있는 객체가 있을텐데, 그 객체를 이용해 계좌 개수를 알아내도 괜찮을 것 같습니다
위 변수는 필요가 없을 것 같습니다